packages/data/src/world/oot/overworld.yml

```
@@ -2314,7 +2315,7 @@
     BUGS: "is_child && has_bottle"
   locations:
     "Gerudo Valley Crate HP": "is_child || can_longshot || climb_anywhere"
-    "Gerudo Valley Waterfall HP": "true"
+    "Gerudo Valley Waterfall HP": "is_child || can_swim" ##Grab the PoH with Cucco as child
```

We should add a region for being down by the cow. That region would have everything after "is_child" and then all that can be removed from here.
```suggestion
    "Gerudo Valley Waterfall HP": "is_child || can_swim || can_longshot || (has_iron_boots && can_hookshot)" ##Grab the PoH with Cucco as child. The Hookshot can attach to the ladder.
```